Smithers.Serialization/SavedItemType.cs

```diff
@@ -68,6 +68,9 @@ public class SavedItemType
         public static readonly SavedItemType SKELETON = new SavedItemType("Skeleton");
         public static readonly SavedItemType BODY_INDEX = new SavedItemType("BodyIndex");
         public static readonly SavedItemType HD_FACE = new SavedItemType("HDFace");
+        public static readonly SavedItemType DEPTH_TO_CAMERA_MAPPING = new SavedItemType("DepthToCamera");
+        public static readonly SavedItemType IR_RIGHT_IMAGE = new SavedItemType("IR_RIGHT");
+        public static readonly SavedItemType IR_LEFT_IMAGE = new SavedItemType("IR_LEFT");
```

Comment:
Hmm, these types don't quite make sense for Kinect. Can you move these declarations to the consuming code instead?